Assets/MRTK/Providers/WindowsMixedReality/Shared/Definitions/HPMotionControllerDefinition.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    public class HPMotionControllerDefinition
+    {
+        public HPMotionControllerDefinition(IMixedRealityInputSource source, Handedness handedness)
+        {
+            inputSource = source;
+            this.handedness = handedness;
+        }
+
+        protected readonly IMixedRealityInputSource inputSource;
+        protected readonly Handedness handedness;
+
+        /// <inheritdoc />
+        public MixedRealityInteractionMapping[] DefaultLeftHandedInteractions => new[]
+        {
+            new MixedRealityInteractionMapping(0, "Spatial Pointer", AxisType.SixDof, DeviceInputType.SpatialPointer),
+            new MixedRealityInteractionMapping(1, "Spatial Grip", AxisType.SixDof, DeviceInputType.SpatialGrip),
+            new MixedRealityInteractionMapping(2, "Grip Position", AxisType.SingleAxis, DeviceInputType.Grip),
+            new MixedRealityInteractionMapping(3, "Grip Touch", AxisType.Digital, DeviceInputType.GripTouch),
+            new MixedRealityInteractionMapping(4, "Grip Press", AxisType.Digital, DeviceInputType.GripPress),
```

Comment:
I think for compatilibity with existing actions and the underlying data providers, this might need to be `AxisType.SingleAxis`.

Reply:
It makes way more sense for it to be `Digital`, but unfortunately the initial implementation had it as `SingleAxis` and it'd be a compatibility issue if we change it now